### ISSUE TITLE
Floyd County 2018 General Election Results have been incorrectly transcribed for SOS & US Senator

### ISSUE THREAD

**Issue by cstewartiii** [openelections/openelections-data-in#72]
The Floyd County 2018 General Election results have been incorrectly transcribed for SOS and US Senator.  This is clear by comparing 20181106__in__general__floyd__precinct.csv with the Floyd County election results, available here:  https://floydcountyclerk.org/index.php/floyd-county-indiana-elections/floyd-county-indiana-election-results/download-file?path=2018-General-Election-Official-Results.pdf.

The issue can be seen in Franklin 1, US Senator.  According to the Floyd County document the votes in the "total votes" column for "normal," "absentee," and "early voting" are 321, 26, and 220, respectively.  In the openelections dataset, these are entered as "votes" for Mike Braun, Joe Donnelly, and Lucy Brenton, respectively.  

In short, the voting mode and candidate totals appear to have been transposed in the U.S. Senate and Secretary of State races.

All of the other data seem to have been entered correctly.
